### PR TITLE
Improve filtering by applicants in bounties explorer

### PR DESCRIPTION
**Pull request by rafalkowalski**
@danlipert 

##### Description

Instead of count method i've added to filter isnull condition.

##### Refers/Fixes

https://github.com/gitcoinco/web/pull/4613/files#r302243404

### CODE REVIEW

File: app/dashboard/router.py [hunk 1]
```diff
@@ -271,13 +271,11 @@ def get_queryset(self):
...
         if applicants == '0':
-            queryset = queryset.annotate(
-                interested_count=Count("interested")
-            ).filter(interested_count=0)
+            queryset = queryset.filter(interested__isnull=True)
         elif applicants == '1-5':
             queryset = queryset.annotate(
                 interested_count=Count("interested")
-            ).filter(interested_count__gte=1).filter(interested_count__lte=5)
+            ).filter(interested_count__gte=1).filter(interested_count__lte=5).cache()
```
Comment (danlipert) [inline app/dashboard/router.py]: what is `.cache()` here?

Comment (thelostone-mc) [reply app/dashboard/router.py]: @rafalkowalski ^

**Comment by thelostone-mc** [comment]
@rafalkowalski  could you address the comment left by @danlipert

**Comment by thelostone-mc** [comment]
@rafalkowalski would you be able to address @danlipert question ? 
I can get this one in once it's approved. If not -> we can close this out and we can revisit it whenever you get back to it

**Pull request closed without merging**